File: 4. SumOfArray.js
let SumOfArray = (arr)=>{
    return arr.reduce((a,b)=>Number(a)+Number(b),0)
}

const readline = require("readline").createInterface({
    input: process.stdin,
    output: process.stdout,
  });
  
  readline.on(`line`, (input) => {
    console.log(SumOfArray([...input.split(" ")]))
    readline.close();
  });
  

File: 4. SumOfArray.cpp
#include<bits/stdc++.h>
using namespace std;
int SumOfArray(int *arr , int n){
    if(n==0)
    return 0;
    return arr[0] + SumOfArray(arr+1,n-1);
}
int main(){
    int n;
    cin>>n;
    int *arr = new int[n];
    cout<<"Sum of arr is "<<SumOfArray(arr,n);
}

